Makefile

```diff
@@ -8,6 +8,7 @@ PROJECTS_PYCODESTYLE="sdk cli toolchain"
 BUILD_PARAMS=
 LEGION_VERSION=0.11.0
```

Comment:
Consider to support logging as admin in Grafana service. Because as a developer, I want to create\edit a dashboard during development.

Reply:
restored